src/app/lib/providers/watchlist.js

```diff
@@ -183,26 +183,7 @@
     };
 
     Watchlist.prototype.detail = function (id, data) {
-        var formatted = {
-            genre: data.genres,
-            synopsis: data.overview,
-        };
-
-        if (data.type === 'movie') {
-            formatted.torrents = {};
-        } else {
-            formatted.episodes = [{
-                episode: data.episode,
-                season: data.season,
-                title: data.episode_title,
-                tvdb_id: data.episode_id,
-                first_aired: new Date(data.episode_aired) / 1000,
-                date_based: false,
-                torrents: {},
-            }];
-        }
-
-        return Promise.resolve(formatted);
+        return Promise.reject('No details for watchlist');
```

Comment:
I think it lacks a few mandatory things (torrents, type), the key names arent the same (genres vs genre, etc), plus it doesnt work for series as there is only the "next episode" in watchlist, not the entire show+episodes list.